tensorflow-examples/src/main/java/org/tensorflow/model/examples/cnn/fastrcnn/FasterRcnnInception.java

```diff
@@ -256,21 +259,18 @@ public static void main(String[] params) {
             DecodeJpeg decodeImage = tf.image.decodeJpeg(readFile.contents(), options);
             //fetch image from file
             try (TUint8 outputImage = (TUint8) runner.fetch(decodeImage).run().get(0)) {
```

Comment:
`outputImage` seems unnecessary now? It's only used for it's shape in lines 337-339, but you've got `imageShape` here at line 262.

Reply:
Used imageShape in lines 337-339